src/calculateRentalCost.js

```diff
@@ -5,6 +5,21 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return price * days;
+  }
+
+  if (days >= 3 && days < 7) {
+    return price * days - smallDiscount;
+  }
+
+  if (days >= 7) {
```

Comment:
The final `if` condition is unnecessary because at this point in the code, it is guaranteed that `days` is 7 or more due to the earlier return statements. You can simply return the calculation without the `if`.